lib/pure/complex.nim

```diff
@@ -376,6 +376,15 @@ proc `$`*(z: Complex): string =
 
 {.pop.}
 
+proc FFT*(dataSource : seq[float]):seq[Complex]= 
```

Comment:
In addition:

* No space before ``:``.
* A space after ``:`` (before the return type).
* A space before the ``=``.